<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgSimpleFilter_binary</class>
 <widget class="QDialog" name="CQDlgSimpleFilter_binary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Binary Filter and Trigger</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QDialogButtonBox" name="qqOkCancel">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>430</y>
     <width>351</width>
     <height>32</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Threshold</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qqThreshold">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="qqTriggerEnabled">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>361</width>
     <height>311</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="title">
    <string>Trigger enabled, triggering conditions</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <widget class="QCheckBox" name="qqVisualize">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>231</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Visualize center of mass and orientation</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>'true' proportion</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLineEdit" name="qqProportion">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>56</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Proportion</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="qqProportionTol">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>20</y>
       <width>56</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>101</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Tolerance (+/-)</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>341</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Center of mass</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>101</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Tolerance (+/-)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="qqPosX">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>56</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="qqPosXTol">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>20</y>
       <width>56</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>X-position</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>101</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Tolerance (+/-)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="qqPosY">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>56</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>91</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Y-position</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="qqPosYTol">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>40</y>
       <width>56</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Orientation</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>101</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Tolerance (+/-)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="qqAngle">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>56</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Angle [deg]</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="qqAngleTol">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>20</y>
       <width>56</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Roundness</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLineEdit" name="qqRoundness">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>56</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Threshold</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>361</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>API return values</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>(1) proportion, (2) center of mass x, (3) center of mass y,
(4) orientation, (5) roundness</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>qqThreshold</tabstop>
  <tabstop>qqTriggerEnabled</tabstop>
  <tabstop>qqVisualize</tabstop>
  <tabstop>qqProportion</tabstop>
  <tabstop>qqProportionTol</tabstop>
  <tabstop>qqPosX</tabstop>
  <tabstop>qqPosXTol</tabstop>
  <tabstop>qqPosY</tabstop>
  <tabstop>qqPosYTol</tabstop>
  <tabstop>qqAngle</tabstop>
  <tabstop>qqAngleTol</tabstop>
  <tabstop>qqRoundness</tabstop>
  <tabstop>qqOkCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
